unlock-app/src/pages/_app.js

```diff
@@ -155,9 +156,11 @@ The Unlock team
                 <Web3ServiceProvider value={web3Service}>
                   <WalletServiceProvider value={walletService}>
                     <ConfigProvider value={config}>
-                      <GlobalErrorConsumer>
-                        <Component {...pageProps} />
-                      </GlobalErrorConsumer>
+                      <CheckoutStoreProvider>
```

Comment:
Would it make more sense to put that one level down, inside of a `FullCheckoutWrapper` component?

Reply:
or maybe actually the existing `CheckoutContainer` ?

Reply:
so that we keep the `App` stuff only for things which are "global"?